tartiflette_aiohttp/_handler.py

```diff
@@ -28,6 +28,17 @@ class BadRequestError(Exception):
     pass
 
 
+class NullAsyncContextManager:
```

Comment:
Trivial: I'm being picky but this isn't really a null context manager because it's awaiting the argument. Would `AsyncAwaitContextManager` be more accurate?

Reply:
This behaves in the same way as https://docs.python.org/3/library/contextlib.html#contextlib.nullcontext
It's named nullcontext in the standard lib, so I opted to name it equivalently here.